exercise3.js

```diff
@@ -46,6 +53,14 @@ hello = () => {
 	return [arr, bigArr]
```

Comment:
why are you returning the original and the bigArr in another array?  the instructions were to return an array with the values squared. and it was supposed to be using an array method.

Reply:
and the name of this function is unclear to me.  why would this function be called "hello" ?